src/oemof/solph/constraints.py

```diff
@@ -326,3 +326,41 @@ def equate_variables(model, var1, var2, factor1=1, name=None):
     def equate_variables_rule(m):
         return var1 * factor1 == var2
     setattr(model, name, po.Constraint(rule=equate_variables_rule))
+
+
+def shared_limit(model, quantity, limit_name,
+                 components, weights, lower_limit=0, upper_limit=None):
+    r"""
+    Adds a constraint to the given model that restricts
+    the weighted sum of variables to a corridor.
+
+    **The following constraints are build:**
+
+      .. math::
+        l_\mathrm{low} \le \sum v_i(t) \times w_i(t) \le l_\mathrm{up}
+        \forall t
+    """
```

Comment:
The docstring should contain more information:

Parameters:
----------------
model....

Examples
------------
....